docs/test-coverage-dashboard.css

```
@@ -0,0 +1,758 @@
+    --status-ok-bg: #e8f5ec;
+    --status-ok-border: #4caf50;
+    --status-partial: #7a5800;
+    --status-partial-bg: #fff8e6;
+    --status-partial-border: #e09000;
+    --status-none: #8b1c1c;
+    --status-none-bg: #fef0f0;
+    --status-none-border: #e53e3e;
+    --priority-high: #c0392b;
+    --priority-medium: #c48a00;
+    --priority-low: #1e8449;
+    --font-mono: 'Courier New', Courier, monospace;
```

Stylelint (17.11.0)

[error] Expected "Courier" to be "courier" (value-keyword-case)

(value-keyword-case)